--- matplotlib-2.1.1/lib/matplotlib/__init__.py.omv~	2017-12-11 22:44:38.900029143 +0100
+++ matplotlib-2.1.1/lib/matplotlib/__init__.py	2017-12-11 22:46:21.830234039 +0100
@@ -633,6 +633,9 @@ def _get_data_path():
                                'directory')
         return path
 
+    if os.path.isdir('/usr/share/matplotlib/mpl-data'):
+        return '/usr/share/matplotlib'
+
     _file = _decode_filesystem_path(__file__)
     path = os.sep.join([os.path.dirname(_file), 'mpl-data'])
     if os.path.isdir(path):
