--- matplotlib-1.4.0/setupext.py.orig	2014-10-13 06:38:28.625252442 +0100
+++ matplotlib-1.4.0/setupext.py	2014-10-13 06:38:50.055250863 +0100
@@ -828,13 +828,6 @@
     name = 'pycxx'
 
     def check(self):
-        if PY3:
-            # There is no version of PyCXX in the wild that will work
-            # with Python 3.x
-            self.__class__.found_external = False
-            return ("Official versions of PyCXX are not compatible with "
-                    "Python 3.x.  Using local copy")
-
         self.__class__.found_external = True
         old_stdout = sys.stdout
         if PY3:
